app/views/curation_concerns/base/_show_actions.html.erb

```diff
@@ -13,16 +16,18 @@
   <%= link_to 'Export to Zotero', sufia.static_path('zotero'), id: 'zoteroLink', name: 'zotero', class: 'btn btn-default' %>
   <%= link_to 'Export to Mendeley', sufia.static_path('mendeley'), id: 'mendeleyLink', name: 'mendeley', class: 'btn btn-default' %>
 
-  <% if @presenter.display_feature_link? %>
-    <%= link_to "Feature", sufia.featured_work_path(@presenter, format: :json),
-      data: {behavior: 'feature'}, id: 'featureLink', name: 'featureLink', class: 'btn btn-default' %>
-  <% end %>
-
-  <% if @presenter.display_unfeature_link? %>
-    <%= link_to "Unfeature", sufia.featured_work_path(@presenter, format: :json),
-      data: {behavior: 'unfeature'}, id: 'unfeatureLink', name: 'unfeatureLink', class: 'btn btn-default' %>
+  <% if @presenter.model_name.name ==  "GenericWork" %>
```

Comment:
Instead of checking the name of the model (identity), can you check the behavior? So replace this line with:

```ruby
<% if @presenter.respond_to(:display_feature_link?) %
```

That gives us more flexibility down the line, allowing users to generate other curation concerns with their own presenters and display the feature/unfeature links if supported.